examples/jsm/animation/CCDIKSolver.js

```diff
@@ -176,22 +185,28 @@ class CCDIKSolver {
 					if ( c > 1.0 ) c = 1.0;
 
 					const c2 = math.sqrt( 1 - c * c );
-					link.quaternion.set( limitation.x * c2,
-					                     limitation.y * c2,
-					                     limitation.z * c2,
-					                     c );
+					link.quaternion.set(
+						limitation.x * c2,
+						limitation.y * c2,
+						limitation.z * c2,
+						c
+					);
```

Comment:
Please avoid changes to the formatting so it's easier to see what you actually change in the source file.